bridges/.config/lingua.dic

```diff
@@ -65,6 +65,8 @@ isn
 io
 js
 keccak256/M
+keypair/MS
+KeyPair
```

Comment:
This should be code right? So we should not whitelist it, but require it to be `\`KeyPair\`` (working on a shortcut for `cargo-spellcheck` to have this in interactive mode).

Reply:
No this is a **comment** refering to the type called `KeyPair`.